src/main/kotlin/io/stud/forest/book_a_cinema_ticket_app/application/dto/RepertoireDto.kt

```diff
@@ -0,0 +1,43 @@
+package io.stud.forest.book_a_cinema_ticket_app.application.dto
+
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder
+import io.stud.forest.book_a_cinema_ticket_app.domain.entities.Screening
+import io.stud.forest.book_a_cinema_ticket_app.domain.repositories.Id
+import java.time.LocalDateTime
+
+@JsonPropertyOrder("movieTitle", "timetable")
+data class RepertoireDto(
+        val movieTitle: String,
+        val timetable: List<TimeTableItem>,
+) {
+    companion object {
+        fun listOf(screenings: List<Screening>): List<RepertoireDto> =
```

Comment:
Dziwna ta nazwa. listOf sugerowałby, że zrobi listę tego co przekazujesz w parametrze, a tymczasem zwraca zupełnie co innego.

Reply:
Faktycznie miałem z nią problem.  
Jak teraz myślę może zrobić to w ten sposób:
`fun List<Screening>.asDtoList(screenings: List<Screening>): List<RepertoireDto>`